packages/test-utils/src/reporter.ts

```diff
@@ -448,8 +431,18 @@ class ReplayReporter<TRecordingMetadata extends UnstructuredMetadata = Unstructu
       hasFilter: !!this._filter,
     });
 
+    mixpanelAPI.trackEvent("test-suite.begin", {
+      baseId: this._baseId,
+      runTitle: this._runTitle,
+      upload: this._upload,
+      hasFilter: !!this._filter,
+    });
+
     if (!this._apiKey) {
       logger.info("OnTestSuiteBegin:NoApiKey");
+
+      mixpanelAPI.trackEvent("test-suite.no-api-key");
```

Comment:
q: couldn't we use a single naming scheme for both of those "loggers"?

Reply:
Yes we could but I don't know if there's value in doing that. They're tracking different things. Sometimes they'll line up (like here) but often they won't. Standardizing on names is work/effort.